cumulusci/robotframework/tests/pageobjects/pageobjects.robot

```diff
@@ -0,0 +1,100 @@
+*** Settings ***
```

Comment:
Can you update the CircleCI configuration to actually run this test suite? That should help with the coverage.